### ISSUE TITLE
Brave crashes when opening the autoplay settings

### ISSUE THREAD

**Issue by pitsi** [brave/brave-browser#17808]
## Description 
As mentioned in the title, brave crashes and exits completely when clicking on the autoplay settings under brave://settings/content.

## Steps to Reproduce
   1. Go to settings > privacy and security > site and shield settings > additional permissions. 
   2. Click autoplay.
   3. Watch brave crash.

## Actual result:
Brave shouldn't crash and show the relevant submenu.

## Expected result:
Brave crashes and has to be relaunched.

## Reproduces how often: 
Easily

## Brave version (brave://version info)
```
Brave	1.29.76 Chromium: 93.0.4577.58 (Official Build) (64-bit)
Revision	c4410ece044414ea42fa4ba328d08195e818a99c-refs/branch-heads/4577@{#1076}
OS	Linux
```

## Version/Channel Information:
- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? No idea
- Can you reproduce this issue with the nightly channel? No idea

## Other Additional Information:
- Does the issue resolve itself when disabling Brave Shields? Not related
- Does the issue resolve itself when disabling Brave Rewards? Not related
- Is the issue reproducible on the latest version of Chrome? Not reproducible because chrome lacks the autoplay as a setting in there.

## Miscellaneous Information:
None.

**Comment by pitsi** [comment]
Also confirmed on a w10 21h1 x64 installation of a friend.

**Comment by Tonev** [comment]
Confirmed the issue in all `Stable`, `Beta`, and `Nightly`.

<html>
<body>


Brave | 1.29.76 Chromium: 93.0.4577.58 (Official Build) (64-bit)
-- | --
Revision | c4410ece044414ea42fa4ba328d08195e818a99c-refs/branch-heads/4577@{#1076}
OS | Windows 7 Service Pack 1 (Build 7601)


</body>
</html>

<html>
<body>


Brave | 1.30.57 Chromium: 93.0.4577.58 (Official Build) beta (64-bit)
-- | --
Revision | c4410ece044414ea42fa4ba328d08195e818a99c-refs/branch-heads/4577@{#1076}
OS | Windows 7 Service Pack 1 (Build 7601)


</body>
</html>

<html>
<body>


Brave | 1.31.10 Chromium: 93.0.4577.58 (Official Build) nightly (64-bit)
-- | --
Revision | c4410ece044414ea42fa4ba328d08195e818a99c-refs/branch-heads/4577@{#1076}
OS | Windows 7 Service Pack 1 (Build 7601)


</body>
</html>

Crash ID's:

`Stable` -> `dfa36d58-6a5b-4a10-ba0d-f5dc66e79b5b`
`Beta` -> `d9ef3351-8bac-4b9c-95fe-e4d26884edf1`
`Nightly` -> `a53a36d2-ae81-4e3c-b2b1-4528c969d283`

**Comment by onchov** [comment]
Confirmed for:
```
Brave | 1.29.76 Chromium: 93.0.4577.58 (Official Build) (64-bit)
Revision | c4410ece044414ea42fa4ba328d08195e818a99c-refs/branch-heads/4577@{#1076}
OS | Linux
```
on 3 different Ubuntu-based machines.

**Comment by Tonev** [comment]
This issue is present in the latest update. I had hopes the Chromium upgrade might resolve the issue.

<html>
<body>


Brave | 1.29.77 Chromium: 93.0.4577.63 (Official Build) (64-bit)
-- | --
Revision | ff5c0da2ec0adeaed5550e6c7e98417dac77d98a-refs/branch-heads/4577@{#1135}
OS | Windows 7 Service Pack 1 (Build 7601)


</body>
</html>

CC: @bsclifton @rebron @iefremov

**Comment by Brave-Matt** [comment]
+1 from Community: https://community.brave.com/t/brave-browser-crashes-when-trying-to-turn-off-autoplay-in-settings/279396/2

**Comment by donbt09** [comment]
I can confirmed Brave crashes when trying to turn off autoplay in settings 
I'm running the latest stable version 1.29.77 Chromium: 93.0.4577.63 (Official Build) (64-bit) on Windows 10 21H1

**Comment by rebron** [comment]
cc: @darkdh May need some help with this one.

**Comment by pitsi** [comment]
If it helps, this is what brave outputs to the terminal the moment it crashes
`/usr/bin/brave-browser: line 48:  5252 Trace/breakpoint trap   "$HERE/brave" "$@"`
I also upgraded to 1.29.77.

**Comment by mac-repo-001** [comment]
Confirmed on Windows 10 21H1 when trying to set Autoplay after first setup
Version 1.29.77 Chromium: 93.0.4577.63 (Official Build) (64-bit)

**Comment by Brave-Matt** [comment]
+1 from Community: 
https://community.brave.com/t/brave-browser-crashes-when-trying-to-turn-off-autoplay-in-settings/279396/4?u=mattches

**Comment by darylsonnier** [comment]
Crash still happening in Brave Version 1.29.79 Chromium: 93.0.4577.63 (Official Build) (64-bit)
Windows 10 Pro 21H1, build 19043.1165/Windows Feature Experience Pack 120.2212.3530.0.

**Comment by pitsi** [comment]
It still crashes on 1.29.79, so why was this closed? Can anyone on beta/dev/nightly confirm it was fixed for them?
```
Brave	1.29.79 Chromium: 93.0.4577.63 (Official Build) (64-bit)
Revision	ff5c0da2ec0adeaed5550e6c7e98417dac77d98a-refs/branch-heads/4577@{#1135}
OS	Linux
```

**Comment by yrliou** [comment]
@pitsi @darylsonnier We close issues when they landed on nightly, uplifts to beta and release are created at https://github.com/brave/brave-core/pull/10013 and https://github.com/brave/brave-core/pull/10014, which is not yet landed. We'll update the issue milestone when it landed.

**Comment by pitsi** [comment]
Thank you for the clarification. However, on another issue I had opened a few months ago (#16251), the bug was closed when the change reached stable.

**Comment by Tonev** [comment]
Your other Github ticket was also closed on the same day the pull request was merged into `Nightly`. Github allows developers to use special words to link an issue to a pull request and close the issue once the pull request is merged, you can take a look over here - https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue.

**Comment by iefremov** [comment]
does anyone have crash ids (aka `Uploaded Crash Report ID:` on brave://crashes)  or backtrace URLs? I'd like to link dumps in backtrace to this issue @darkdh @Brave-Matt

**Comment by darylsonnier** [comment]
@iefremov, I can provide more if you'd like, but these are the three most recent crash IDs.  

00e9067a-074c-4534-9b64-421f749a29fc

379eed82-3669-4e79-b4a7-ddf5feac7df9

8651aaf6-35a2-466f-a0fe-ec4e479cedd3

**Comment by Tonev** [comment]
@iefremov 

I provided some crash ID's earlier, believe I even sent via `brave://crashes`.

https://github.com/brave/brave-browser/issues/17808#issuecomment-910482505

**Comment by iefremov** [comment]
Thanks a lot @darylsonnier @Tonev , that helped!

**Comment by GeetaSarvadnya** [comment]
Verification passed on

```
Brave | 1.29.80 Chromium: 93.0.4577.63 (Official Build) (64-bit)
-- | --
Revision | ff5c0da2ec0adeaed5550e6c7e98417dac77d98a-refs/branch-heads/4577@{#1135}
OS | Windows 10 OS Version 2009 (Build 19043.1165)

```
- Verified the STR from the description
- Reproduced the issue in the version `1.29.79`
- Confirmed issue is fixed and working fine in `1.29.80`

**Comment by Tonev** [comment]
@kjozwiak 

Please take a look when possible as you've verified on `MacOS`.

https://community.brave.com/t/browser-crashes-when-opening-autoplay-setting/279706/4?u=rethanis
